storage/src/vespa/storage/config/stor-communicationmanager.def

```diff
@@ -68,6 +68,8 @@ rpc.num_network_threads int default=1
 ## The bucket id associated with a message is used to select the RPC target.
 ## This ensures the same RPC target is used for all messages to the same bucket to the same node,
 ## and the RPC target itself handles sequencing of these messages.
+## NB !! It is vital that this number is kept in sync with stor-filestor:num_network_threads.
+## Only skilled vespa core devolpers should touch this.
```

Comment:
Typo: devolpers -> developers

Reply:
Fixed in https://github.com/vespa-engine/vespa/pull/14970